plugins/modules/imgadm.py

```diff
@@ -234,8 +234,7 @@ def manage_images(self):
                 else:
                     self.changed = True
         if self.present:
-            cmd = '{0} import -P {1} -q {2}'.format(self.cmd, pool, self.uuid)
-
+            cmd = [self.cmd, 'import', '-P', pool, '-q'] + [self.uuid] if self.uuid else []
```

Comment:
```suggestion
            cmd = [self.cmd, 'import', '-P', pool, '-q'] + ([self.uuid] if self.uuid else [])
```
(The original form is equivalent to `cmd = [self.cmd, 'import', '-P', pool, '-q', self.uuid] if self.uuid else []`.)